CHANGELOG.md

```diff
@@ -325,6 +325,19 @@ The following rules are now recommended:
   type NonNullableMyType = MyType & {};
   ```
 
+- Improve [noConstantCondition](https://biomejs.dev/lint/rules/noConstantCondition/)
+
+  The rule now allows `while(true)`.
+  This recognizes a common pattern in the web community:
+
+  ```js
+  while (true) {
+    if (cond) {
+      break;
+    }
+  }
+  ```
```

Comment:
Shouldn't we track the code action too?

Reply:
What do you mean?

Reply:
I mean the rule now has a code action, and the changelog doesn't say anything about it

Reply:
This is a mistake. This should be part of another PR. Thanks for bringing it to light!